clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/KafkaConsumerServiceTest.java

```diff
@@ -202,26 +216,29 @@ public void testTopicWiseGetConsumerForHybridMode() throws Exception {
 
   @Test
   public void testPartitionWiseGetConsumer() throws Exception {
-    SharedKafkaConsumer consumer1 = mock(SharedKafkaConsumer.class);
+    ApacheKafkaConsumer consumer1 = mock(ApacheKafkaConsumer.class);
```

Comment:
For here, `KafkaClientFactory#getConsumer(...)` just return `ApacheKafkaConsumer`, the method return type is `PubSubConsumer`, so it can be `SharedKafkaConsumer`. So just want to emulate the real case.